src/main/java/org/commcare/formplayer/session/MenuSession.java

```diff
@@ -302,9 +355,9 @@ private void computeDatum() {
         EvaluationContext ec = sessionWrapper.getEvaluationContext();
         if (datum instanceof FormIdDatum) {
```

Comment:
aside: What is a `FormIdDatum`? I can't find any such thing in HQ.

Reply:
I am not aware of it's use case, probably a legacy thing @ctsims  ?

Reply:
This datum lets you configure an XMLNS dynamically based on session data. I think it's a legacy thing that we haven't used for a long time, and I'm not sure if HQ ever let you configure it, it might have been a remote-apps only thing.